orbstation/clothing/wintercoats.dm

```diff
@@ -0,0 +1,85 @@
+//Winter coats for several additional jobs, imported from Skyrat
+
+//Bartender
+/obj/item/clothing/suit/hooded/wintercoat/bartender
+	icon = 'orbstation/icons/obj/clothing/suits.dmi'
+	worn_icon = 'orbstation/icons/mob/clothing/suit.dmi'
+	name = "bartender's winter coat"
+	desc = "A heavy jacket made from wool originally stolen from the chef's goat. This new design is made to fit the classic suit-and-tie aesthetic, but without the hypothermia."
+	icon_state = "coatbar"
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
+	allowed = list(
+		/obj/item/flashlight,
+		/obj/item/tank/internals/emergency_oxygen,
+		/obj/item/tank/internals/plasmaman,
+		/obj/item/toy,
+		/obj/item/storage/fancy/cigarettes,
+		/obj/item/lighter,
+		/obj/item/reagent_containers/cup/glass/shaker,
+		/obj/item/reagent_containers/cup/glass/flask,
+		/obj/item/reagent_containers/cup/rag,
+		/obj/item/gun/ballistic/shotgun/doublebarrel,)
```

Comment:
Does the bartender usually have a suit they can put the shotgun in? I cannot find another one in the code.
That seems like a pretty substantial benefit they don't usually have.

Reply:
The armor vest the bartender spawns with can hold the shotgun (and probably any guns, actually?). I thought this would help make the coat an appealing option even if it isn't armored.

Though maybe this coat _should_ be slightly armored because of that...